includes/ucf-spotlight-common.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Place common functions here.
+ **/
+
+if ( ! class_exists( 'UCF_Spotlight_Common' ) ) {
+	class UCF_Spotlight_Common {
+
+		public function enqueue_styles() {
+			if ( get_option( 'ucf_spotlight_include_css' ) ) {
+				wp_enqueue_style( 'ucf_spotlight_css', plugins_url( 'static/css/ucf-spotlight.min.css', UCF_SPOTLIGHT__PLUGIN_FILE ), false, false, 'all' );
+			}
+		}
+
+		public static function display_spotlight( $attr ) {
+			$output = ucf_spotlight_display( $attr );
+			return apply_filters( 'ucf_spotlight_display', $output );
+		}
+	}
+
+	add_action( 'wp_enqueue_scripts', array( 'UCF_Spotlight_Common', 'enqueue_styles' ), 99 );
```

Comment:
Any particular reason for using 99 as the priority level?  I think this _might_ pose problems for folks using the plugin who want to enqueue an additional stylesheet before or after the plugin's CSS by setting 'ucf_spotlight_css' as a dependent stylesheet--unless they can use an action priority greater than 99, I don't think they'll be able to time it correctly.